client/src/resources/cert-and-project-map.ts

```diff
@@ -760,6 +761,20 @@ const certMap = [
 ] as const;
 
 const upcomingCertMap = [
+  {
+    id: '647e3159823e0ef219c7359b',
+    title: 'Foundational C# with Microsoft',
+    certSlug: 'foundational-c-sharp-with-microsoft',
+    flag: 'isFoundationalCSharpCert',
```

Comment:
Again, not a problem with this PR, but any idea what this is for? I can't find any references to `flag` elsewhere.

Reply:
Nope. Again, just following the structure that was already in place. I would just leave them for this PR. We could dig into these and see if we can delete them.